From 784691dd2eceba4128b6dd5744677e82053ad019 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Fri, 30 Dec 2011 18:10:49 -0500
Subject: [PATCH 1/2] Milestone4.0

---
 WHAT_IS_INSIDE |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/WHAT_IS_INSIDE b/WHAT_IS_INSIDE
index 3c5ecef..0346f07 100644
--- a/WHAT_IS_INSIDE
+++ b/WHAT_IS_INSIDE
@@ -21,6 +21,8 @@ Milestone2.0: Mejoras en Instances y dungeons, escencialmente para darles funcio
               a blizzlike, solucion de bugs varios.
 Milestone3.0  Arreglos a Spells, sobre todo para estabilidad del pve y del
               pvp dentro de los reinos.
+Milestone4.0  Arreglos en quests, eventos y cadenas de mision de facciones
+              immportantes, esto incluye scripts en c++ y SmartAI.
 
 
 ##################################
@@ -198,6 +200,7 @@ Milestone3.0  Arreglos a Spells, sobre todo para estabilidad del pve y del
 ############################
 # Soporte Quests y Eventos #
 ############################
+4.1.1  - 20/12/2011: WinterVeil: Añadidos soporte para logros faltantes por Retriman
 
 
 #################
-- 
1.7.6.msysgit.0

