From 5beab1f278c053d207ce69208c1b179b9fa10d36 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Mon, 26 Dec 2011 12:49:30 -0500
Subject: [PATCH 1/2] Milestone5.0

---
 WHAT_IS_INSIDE |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/WHAT_IS_INSIDE b/WHAT_IS_INSIDE
index 8845e68..1ffdbc2 100644
--- a/WHAT_IS_INSIDE
+++ b/WHAT_IS_INSIDE
@@ -23,6 +23,8 @@ Milestone3.0  Arreglos a Spells, sobre todo para estabilidad del pve y del
               pvp dentro de los reinos.
 Milestone4.0  Arreglos en quests, y cadenas de mision, esto incluye scripts
               en c++ y SmartAI.
+Milestone5.0  Traducciones, e inclusion de varios dentro de tablas y strings
+              del emulador, ya sea en c++ o en sqls.
 
 
 ##################################
@@ -205,6 +207,7 @@ Milestone4.0  Arreglos en quests, y cadenas de mision, esto incluye scripts
 #################
 # Traducciones: #
 #################
+5.1  - 03/06/2011: Traducciones de eswow2 de varias tablas de world, no he añadido todas porque crashean por Eilo
 
 
 ##########
-- 
1.7.6.msysgit.0

