From 89305daaa1cc4e905feedcd5b71fffa7c734d6bb Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Wed, 28 Dec 2011 14:34:03 -0500
Subject: [PATCH] Actualizado con Trinitycore al 15/12/2011

---
 WHAT_IS_INSIDE |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/WHAT_IS_INSIDE b/WHAT_IS_INSIDE
index e0099c1..dd0be20 100644
--- a/WHAT_IS_INSIDE
+++ b/WHAT_IS_INSIDE
@@ -31,6 +31,8 @@ Milestone6.0  Customs y varios
 ##################################
 # Trintitycore hasta esta fecha: #
 ##################################
+15/12/2011: Commit:af5474c959b5706e24230ce9c45f12314711501b
+* Core/Petitions: Fix a typo in a sql query (d59c5e627fd023d3)
 
 
 ###############
-- 
1.7.6.msysgit.0

