From 334708c6f17fe37c52ba8e9515ee6bfc1016e42f Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Tue, 13 Dec 2011 23:02:31 -0500
Subject: [PATCH 04/14] Milestone1.0

---
 WHAT_IS_INSIDE |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/WHAT_IS_INSIDE b/WHAT_IS_INSIDE
index 6439e90..5f6cc62 100644
--- a/WHAT_IS_INSIDE
+++ b/WHAT_IS_INSIDE
@@ -13,6 +13,8 @@ Milestone0.0: Reiniciando el repo y readaptando cosas varias, inclusion de nuevo
               developers en git y la inclusion de texto de planeacion para union 
 			  eficiente de codigo reestructurando modo de trabajo y organizando 
 			  carpetas, branches, etc
+Milestone1.0: Mejoras en general, aplicacion de planificacion y adicion de codigo
+              utilitario en general para complemento del nucleo.
 
 			  
 ##################################
-- 
1.7.6.msysgit.0

