From b0ee0ee5b3e2894498645dc3d1967eff66a703fc Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Fri, 16 Dec 2011 02:37:13 -0500
Subject: [PATCH 45/77] 2.2.17 ICC: Saurfang aggrea apenas termina el dialogo

---
 .../IcecrownCitadel/boss_deathbringer_saurfang.cpp |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/server/scripts/Northrend/IcecrownCitadel/boss_deathbringer_saurfang.cpp b/src/server/scripts/Northrend/IcecrownCitadel/boss_deathbringer_saurfang.cpp
index c02a04b..9be6bf1 100755
--- a/src/server/scripts/Northrend/IcecrownCitadel/boss_deathbringer_saurfang.cpp
+++ b/src/server/scripts/Northrend/IcecrownCitadel/boss_deathbringer_saurfang.cpp
@@ -253,7 +253,6 @@ class boss_deathbringer_saurfang : public CreatureScript
             void Reset()
             {
                 _Reset();
-                me->SetReactState(REACT_DEFENSIVE);
                 events.SetPhase(PHASE_COMBAT);
                 _frenzied = false;
                 me->SetPower(POWER_ENERGY, 0);
@@ -437,6 +436,8 @@ class boss_deathbringer_saurfang : public CreatureScript
                             events.SetPhase(PHASE_COMBAT);
                             _introDone = true;
                             me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_OOC_NOT_ATTACKABLE);
+                            me->SetReactState(REACT_AGGRESSIVE);
+                            me->SetInCombatWithZone();
                             break;
                         case EVENT_SUMMON_BLOOD_BEAST:
                             for (uint32 i10 = 0; i10 < 2; ++i10)
-- 
1.7.6.msysgit.0

