From e203cca8b7f3f758ee4a272c226ee0252e72d2c4 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Sat, 17 Dec 2011 14:42:52 -0500
Subject: [PATCH 76/77] 2.14.2 Drak'Tharon: Novos permitir reinicio boss

---
 .../Northrend/DraktharonKeep/boss_novos.cpp        |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/server/scripts/Northrend/DraktharonKeep/boss_novos.cpp b/src/server/scripts/Northrend/DraktharonKeep/boss_novos.cpp
index e877126..0a6c51b 100644
--- a/src/server/scripts/Northrend/DraktharonKeep/boss_novos.cpp
+++ b/src/server/scripts/Northrend/DraktharonKeep/boss_novos.cpp
@@ -143,6 +143,9 @@ public:
 
         void UpdateAI(const uint32 diff)
         {
+            if(!UpdateVictim())
+                return;
+
             switch (Phase)
             {
                 case PHASE_1:
-- 
1.7.6.msysgit.0

