From 222fda8c53e285cb42428362e773810dda1be5f0 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Mon, 2 Jan 2012 12:01:22 -0500
Subject: [PATCH 19/19] Milestone1.0

---
 WHAT_IS_INSIDE |   19 ++++++++++++++++++++
 1 files changed, 19 insertions(+), 0 deletions(-)

diff --git a/WHAT_IS_INSIDE b/WHAT_IS_INSIDE
index 58934e8..8153fe0 100644
--- a/WHAT_IS_INSIDE
+++ b/WHAT_IS_INSIDE
@@ -14,6 +14,8 @@ Milestone0.0: Reiniciando el repo y readaptando cosas varias, inclusion de nuevo
               eficiente de codigo reestructurando modo de trabajo y organizando 
               carpetas, branches, etc, adicion de codigo utilitario en general 
               para complemento del nucleo y funciones de soporte.
+Milestone1.0: Jugabilidad en general, adecuaciones de codigo vario en mapas
+              comportamiento de bgs, arenas, players-units-pets y mejoras.
 
 
 ##################################
@@ -53,6 +55,24 @@ Milestone0.0: Reiniciando el repo y readaptando cosas varias, inclusion de nuevo
 ###############
 # Jugabilidad #
 ###############
+1.1  - 18/10/2011: Inmunidades de vehiculos por Karlos
+1.2  - 26/09/2011: Prohibicion de vehiculos en arenas por Snajl
+1.3  - 18/10/2011: Reseteo de talentos duales a no duales por Faelon
+1.4  - 27/10/2011: Bindeo instance players que se salen y lotean por Greymane
+1.5  - 18/10/2011: Apaño Parry formula de calculo por Faelon
+1.6  - 18/10/2011: Apaño del Dodge formula y comportamiento por Faelon
+1.7  - 22/10/2011: Guardado de Spells, evitando bugs especialmente del dk por Odyssey
+1.8  - 11/05/2011: EyeOfStorm: Fix entrega de bandera en FelReaver(AtracadorVil) por Tobmaps actualizado por Eilo
+1.9  - 30/10/2011: WarsongGlutch: Mejora en drop de ambas banderas a cercania por Resolver10
+1.10 - 10/10/2011: DynamicLoS complementos y activacion en la db, ajuste circulo demoniaco en estas 2 arenas por Karlos
+1.11 - 13/01/2012: Prevenir exploit login dentro de instances por Tobmaps
+1.12 - 03/11/2011: Mejora en pesos(probabilidad) de cada bg y min-max pj por Eilo
+1.13 - 15/10/2011: Deshabilitadas spells de nefarian que usan para exploit por Havenard
+1.14 - 29/10/2011: Evitando que puedan poner totems dentro de las paredes por Opcode187
+1.15 - 20/12/2011: Mejoras en altura(z) criaturas que vuelan o nadan por Zwerg
+1.16 - 05/11/2011: Prevenir error en el cálculo de buffs con stacks por Kerhong
+1.17 - 17/12/2011: IsleOfConquest: Script evitando que puedan matar a los bosses con exploit por Retriman
+1.18 - 08/11/2011: ArPen mejora comportamiento, mas blizzlike por Ws
 
 
 ###############################
-- 
1.7.8.msysgit.0

